### PR TITLE
Tickets/cap 376

### PR DESCRIPTION
**Pull request by tony-johnson**
Starter set of ComCam commands/events/telemetry. Should be included in ts_xml 4.6 release.
Note pytest was riun, and appeared to pass except for unit torr (see CAP-396)

### CODE REVIEW

File: sal_interfaces/CCCamera/CCCamera_Telemetry.xml [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+    </SALTelemetry>
+    <SALTelemetry>
+        <Subsystem>CCCamera</Subsystem>
+        <Version>3.10.2</Version>
+        <EFDB_Topic>CCCamera_vacuumIonPump</EFDB_Topic>
+        <item>
+            <EFDB_Name>cIP1_V</EFDB_Name>
+            <Description>Cryo Ion pump 1 voltage</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Volt</Units>
```
[the remark this answers is not in the document]
Comment (tony-johnson) [reply sal_interfaces/CCCamera/CCCamera_Telemetry.xml]: Fixed.

File: sal_interfaces/SALSubsystems.xml [hunk 2]
```diff
@@ -595,6 +595,28 @@
   <Simulator></Simulator>
 </SALSubsystem>
 
+<SALSubsystem>
```
[the remark this answers is not in the document]
Comment (tony-johnson) [reply sal_interfaces/SALSubsystems.xml]: More info added, but many fields seem ambiguous/unclear/not applicable.

[the remark this answers is not in the document]
Comment (mareuter) [reply sal_interfaces/SALSubsystems.xml]: Agreed. We need to document what's expected here.

**Comment by rbovill** [comment]
I made a few changes, on your behalf, to pick necessary features (Torr workaround, Reserved Word Jira ticket).  However, there are still two real issues; blank descriptions in the Events and Telemetry XML files.  Those will need to be fixed before the unit tests will pass.

**Comment by tony-johnson** [comment]
All new items now have a description.

File: sal_interfaces/CCCamera/CCCamera_Commands.xml [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+    <EFDB_Topic>CCCamera_command_disableCalibration</EFDB_Topic>
+    <Alias>disableCalibration</Alias>
+    <Device></Device>
+    <Property></Property>
+    <Action></Action>
+    <Value></Value>
+    <Explanation>http://project.lsst.org/ts/sal_objects/help</Explanation>
+    <item>
+        <EFDB_Name>value</EFDB_Name>
+        <Description>Unused</Description>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Commands.xml]: The description for unused parameters is: _Attribute required by the API, but is unused_.

File: sal_interfaces/CCCamera/CCCamera_Commands.xml [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+    <EFDB_Topic>CCCamera_command_enableCalibration</EFDB_Topic>
+    <Alias>enableCalibration</Alias>
+    <Device></Device>
+    <Property></Property>
+    <Action></Action>
+    <Value></Value>
+    <Explanation>http://project.lsst.org/ts/sal_objects/help</Explanation>
+    <item>
+        <EFDB_Name>value</EFDB_Name>
+        <Description>Unused</Description>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Commands.xml]: The description for unused parameters is: _Attribute required by the API, but is unused_.

File: sal_interfaces/CCCamera/CCCamera_Commands.xml [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+    <EFDB_Topic>CCCamera_command_endImage</EFDB_Topic>
+    <Alias>endImage</Alias>
+    <Device></Device>
+    <Property></Property>
+    <Action></Action>
+    <Value></Value>
+    <Explanation>http://project.lsst.org/ts/sal_objects/help</Explanation>
+    <item>
+        <EFDB_Name>value</EFDB_Name>
+        <Description>Unused</Description>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Commands.xml]: The description for unused parameters is: _Attribute required by the API, but is unused_.

Comment (tony-johnson) [reply sal_interfaces/CCCamera/CCCamera_Commands.xml]: Done

File: sal_interfaces/CCCamera/CCCamera_Commands.xml (hunk 4 above)
Comment (tony-johnson) [reply sal_interfaces/CCCamera/CCCamera_Commands.xml]: This is fixed.

File: sal_interfaces/CCCamera/CCCamera_Commands.xml (hunk 3 above)
Comment (tony-johnson) [reply sal_interfaces/CCCamera/CCCamera_Commands.xml]: Fixed

**Pull request merged**